site/SiteTools.tsx

```diff
@@ -16,21 +16,24 @@ const SITE_TOOLS_CLASS = "site-tools"
 
 const SiteTools = () => {
     const scrollDirection = useScrollDirection()
+    const isDonatePage = /^\/donate\/?$/.test(window.location.pathname)
```

Comment:
not wrong per se, but curious about the "?" in the regex, since we redirect /donate/ to /donate anyway?

Reply:
Well, we don't in local env, so I didn't know about it. Removing that case then!